pkg/config/setup/process.go

```diff
@@ -172,7 +172,7 @@ func setupProcesses(config pkgconfigmodel.Setup) {
 		"DD_PROCESS_ADDITIONAL_ENDPOINTS",
 	)
 	procBindEnvAndSetDefault(config, "process_config.events_additional_endpoints", make(map[string][]string))
-	config.SetKnown("process_config.intervals.connections")
+	procBindEnv(config, "process_config.intervals.connections")
```

Comment:
Please add a default to the settings.

Reply:
why?
I'm trying to keep the current code as is

Currently, there's a logic to apply a default interval, unless the config was set https://github.com/DataDog/datadog-agent/blob/004a5befa3bc2b103bb4f3e23cab471ebbd86136/pkg/process/checks/interval.go#L92-L102

Reply:
We're trying to move toward a real schema for the configuration. Settings without defaults and types are a blocker for this. Also relying on post-processing of the config for defaults makes it difficult for users and other devs to understand the behavior of the agent just by looking at the configuration definition.

`IsSet` and `IsKnow` have been deprecated (`IsSet` in itself is very error prone since it only checks if a value exists, not that it's valid). You should instead use the `IsConfigured` method from the config to know if a setting was configured by the user or not.

Reply:
OK, let's do this in another PR then.